src/checks.h

```diff
@@ -0,0 +1,31 @@
+#ifndef _CHECKS_H_
+#define _CHECKS_H_
```

Comment:
`#pragma once`

Reply:
I don't like `#pragma once` as I already had issue with samely-named files (in different directories) and the pragma can't deal with it. It does not apply here, but every other header uses macros and I dislike inconsistency. :shrug: